lib/compiler/build_runner.zig

```diff
@@ -1293,7 +1293,9 @@ fn usage(b: *std.Build, out_stream: anytype) !void {
         \\  -j<N>                        Limit concurrent jobs (default is to use all CPU cores)
         \\  --maxrss <bytes>             Limit memory usage (default is to use available memory)
         \\  --skip-oom-steps             Instead of failing, skip steps that would exceed --maxrss
-        \\  --fetch                      Exit after fetching dependency tree
+        \\  --fetch[=mode]               Fetch dependency tree (optionally choose laziness) and exit
+        \\    lazy                       (Default) Lazy dependencies are fetched on need
```

Comment:
Nit:

```suggestion
        \\    lazy                       (Default) Lazy dependencies are fetched as needed
```

Reply:
Also, what do you think about `needed` instead of `lazy`? I feel like that might go better with `all`.